js/src/launchAsync.ts

```diff
@@ -78,20 +85,34 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             return;
         }
 
+        if (options && options.launchMode) {
+            if (options.launchMode === 'reading' && !options.onReceiveReadingPracticeMessage) {
```

Comment:
shall we add a code comment on each of these code blocks?

Reply:
same code as internal, it is just validating that caller is passing the options needed